sql/core/src/test/resources/sql-tests/inputs/bitwise.sql

```diff
@@ -0,0 +1,44 @@
+-- test cases for bitwise functions
+
+-- null
+
+select bit_count(null);
+
+-- boolean
+select bit_count(true);
+select bit_count(false);
+
+-- byte/tinyint
+select bit_count(cast(1 as tinyint));
+select bit_count(cast(2 as tinyint));
+select bit_count(cast(3 as tinyint));
+
+-- short/smallint
+select bit_count(1S);
+select bit_count(2S);
+select bit_count(3S);
+
+-- int
+select bit_count(1);
+select bit_count(2);
+select bit_count(3);
+
+-- long/bigint
+select bit_count(1L);
+select bit_count(2L);
+select bit_count(3L);
+
+-- negative num
+select bit_count(-1L);
+
+-- mysql> select 9223372036854775807 + 1;
+-- ERROR 1690 (22003): BIGINT value is out of range in '(9223372036854775807 + 1)'
+-- mysql> select bit_count(9223372036854775807 + 1);
+-- ERROR 1690 (22003): BIGINT value is out of range in '(9223372036854775807 + 1)'
+-- out of range
+select 9223372036854775807L + 1L;
```

Comment:
the overflow behavior is nothing to do with the `bit_count` function, or did I miss something?

Reply:
yes, you're right. The case here illustrates the overflow behavior is nothing to do with the `bit_count`, which may cause inconsistency with mysql

Reply:
We have test suites for the overflow behavior. I don't think it makes sense to test overflow again in all the functions that take integers.

Reply:
ok, I will rm these and just check long min/max